counterpartylib/lib/api.py

```diff
@@ -764,6 +779,255 @@ def handle_root(args_path):
                 # Not found
                 return flask.Response(None, 404, mimetype='application/json')
 
+        @dispatcher.add_method
+        def sendrawtransaction(tx_hex):
+            micropayments.validate.tx_signature(dispatcher, tx_hex)
+            return backend.sendrawtransaction(tx_hex)
+
+        #########################
+        # Micropayment channels #
+        #########################
+
+        @dispatcher.add_method
+        def mpc_make_deposit(asset, payer_pubkey, payee_pubkey,
+                             spend_secret_hash, expire_time, quantity):
+            """ Create deposit and setup initial payer state.
```

Comment:
Probably remove these docstrings, as the other API methods don't have them, and it's redundant with what's in`API.md`.

Also, please check your docs PR at https://github.com/CounterpartyXCP/Documentation/pull/123/files and make sure it is up to date, thanks!